Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml

```diff
@@ -1,24 +1,22 @@
 alert_category: EXECUTION
 alert_description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
 alert_fields:
-  actor_process_image_name:
-  actor_process_image_path: xdm.target.file.path
+  action_local_ip: xdm.target.ipv4
+  actor_effective_username: xdm.source.user.username
   actor_process_command_line: xdm.target.process.command_line
-  agent_device_domain: xdm.source.user.domain
+  actor_process_image_path: xdm.target.file.path
   actor_process_image_sha256: xdm.target.file.sha256
+  agent_device_domain: xdm.source.user.domain
   agent_hostname: xdm.target.host.fqdn
-  action_local_ip: xdm.target.ipv4
-  action_remote_ip:
-  action_remote_port:
-  actor_effective_username: xdm.source.user.username
+  eventid: xdm.event.id
 alert_name: Malware detected by Microsoft Defender for Endpoints
 crontab: '*/10 * * * *'
 dataset: alerts
-description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
+description: An alert will trigger in an event when a Malware was detected by Microsoft Defender for Endpoint.
```

Comment:
Sorry for the hassle, but I think the best thing to write is:
This alert will trigger when Malware is detected by Microsoft Defender for Endpoint.
And I would copy the same to alert_description.

Reply:
@OBavly Done!
Ready for re-approval.